.travis.yml

```diff
@@ -7,17 +7,12 @@ python:
 - '3.10'
 - '3.11'
 env:
-- PINT_VERSION=0.18
-- PINT_VERSION=0.21
-jobs:
-  exclude:
-    - python: 3.7
-      env: PINT_VERSION=0.21
+  - UPGRADES="-U pint pandas"
+  - UPGRADES=""
```

Comment:
Is this reassigning `UPGRADES` to the empty string?

Reply:
No, this is saying run once with the upgrades, and once using the versions specified in the requirements files.

https://docs.travis-ci.com/user/languages/python/#testing-against-multiple-versions-of-dependencies-eg-django-or-flask

Reply:
TIL! Thanks for the new knowledge :smile: